lib/rubocop/cop/rails/link_to_blank.rb

```diff
@@ -74,7 +74,9 @@ def append_to_rel(rel_node, corrector)
         def add_rel(send_node, offence_node, corrector)
           quote_style = offence_node.children.last.source[0]
           new_rel_exp = ", rel: #{quote_style}noopener#{quote_style}"
-          corrector.insert_after(send_node.loc.expression, new_rel_exp)
+          range = send_node.children.last.source_range
```

Comment:
[nitpick]
Personally I think `send_node.arguments.last.source_range` is more meaningful. What do you think?

```suggestion
          range = send_node.arguments.last.source_range
```

Reply:
Sure! I updated it.